### ISSUE TITLE
Panel with id not found - Clicking From Alert List Panel - v6.0.0

### ISSUE THREAD

**Issue by APenketh** [grafana/grafana#15680]
### What Grafana version are you using?
v6.0.0

### What datasource are you using?
Seem's to be reproducible with InfluxDB Panels

### What OS are you running grafana on?
CentOS 7.3

### What did you do?
When an alert is generated I click on the alert from the Alert List Panel. At the moment I cannot reproduce it from anywhere else.

### What was the expected result?
Normally it would take you to the Panel where the alert is being triggered. 

### What happened instead?
You are taken to the dashboard where the panel is located however get the following error;

![image](https://user-images.githubusercontent.com/18573773/53460093-ebe95480-3a76-11e9-8a3a-a4a3cbb7f5df.png)

I have tried to delete the panel and re-create however the same behavior exists.

### Include raw network request & response: get by opening Chrome Dev Tools (F12, Ctrl+Shift+I on windows, Cmd+Opt+I on Mac), go the network tab.
 
No errors reported.

**Comment by xlson** [comment]
Could you share your dashboard json so that I can see if something has gone wrong there?

**Comment by dmitryilyin** [comment]
Yes, it happens for me too. If you click alert in alert list to open on the same page the described problem happens, but if you click to open on a new tab it opens normally.

**Comment by Mixajlo14** [comment]
Hi @xlson , we also have this problem. You can find example of this on play grafana dashboard (left click on "Customer Reviews alert" in alert list on next link https://play.grafana.org/d/000000074/alerting?orgId=1). 
This doesn't happen for every alert, only for some of them.

**Comment by torkelo** [comment]
thanks for that link, able to replicate now!
